### PR TITLE
Ammo sprite tweaks

### PR DESCRIPTION
**Pull request by MoXuT0**
## About The Pull Request

Adds unique magazine sprites for M42A, M4RA and original M41A. All standart rifle mags have slight curve now. Also fixes rogue pixels on some empty rifles in `gun.dmi`.

## Changelog

:cl:
imageadd: Added unique magazine sprites for M42A, M4RA and M41A. Slightly tweaked mags for regular and heavy pulse rifles.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/updated_projectiles/magazines/rifles.dm [hunk 1]
```diff
@@ -17,20 +17,20 @@
 /obj/item/ammo_magazine/rifle/extended
 	name = "\improper M41A extended magazine (10x24mm)"
 	desc = "A 10mm assault extended rifle magazine."
-	icon_state = "m41a_ext"
+	icon_state = "m41a1_ext"
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: The name of the gun is `M41A`, why is it being changed to `M41A1` in the icon?

Comment (MoXuT0) [reply code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: It was renamed in #1494 not sure why it doesn’t show up here.

Comment (DominikPanic) [reply code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: Because you haven't rebased likely

Comment (Alterist) [reply code/modules/projectiles/updated_projectiles/magazines/rifles.dm]: i renamed the standard pulse rifle to m41a1, the og is called m41a. everything is correct here

**Pull request merged**